eclass/acct-user.eclass

```diff
@@ -222,22 +222,25 @@ elockuser() {
 	eislocked "$1"
 	[[ $? -eq 0 ]] && return 0
 
+	local opts
+	[[ ! -z ${ROOT} ]] && opts=( -R "${ROOT}" )
```

Comment:
```suggestion
	[[ -n ${ROOT} ]] && opts=( -R "${ROOT}" )
```

and below. Really easier to understand stuff without unnecessary negations.

Reply:
For the netBSD stuff, this is actually processed by the *) condition right ?
In that case, do I need to process the user/group creation with awk ? Or just printing a warning telling that the user/group is actually not created in the target and should be added manually ? (with `chroot <command>` assuming ARCH is compatible with the host system)

Reply:
Just issue a warning, like the one we're already issuing if we can't do stuff or it fails.

Reply:
If you want to be compatibly with both eapi == 7 and eapi != 7, I think you need `[[ -n "${ROOT}" ]] && [[ "${ROOT}" != "/" ]]`

Reply:
@Mord3rca